python/ray/air/checkpoint.py

```diff
@@ -760,12 +760,7 @@ def __fspath__(self):
             "Use `Checkpoint.to_directory()` or `Checkpoint.as_directory()` instead."
         )
 
-    def get_preprocessor(self) -> Optional["Preprocessor"]:
-        """Return the saved preprocessor, if one exists."""
-
-        # The preprocessor will either be stored in an in-memory dict or
-        # written to storage. In either case, it will use the PREPROCESSOR_KEY key.
-
+    def _get_preprocessor(self) -> Optional["Preprocessor"]:
```

Comment:
Do we need this internal function? We don't use it anywhere else?

Reply:
It's used below